tests/bootstrap.php

```diff
@@ -13,10 +13,6 @@
 $loader->add('Jackalope\\Test', __DIR__);
 $loader->add('Jackalope', __DIR__ . '/../vendor/jackalope/jackalope/tests');
```

Comment:
ah, the first one yet, but this one not because the jackalope tests should not be autoloaded for consumers of the library